pkg/common/redis.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/beam-cloud/beta9/pkg/types"
 	"github.com/bsm/redislock"
+	"github.com/mitchellh/mapstructure"
```

Comment:
Can we pull in this one instead? https://pkg.go.dev/github.com/go-viper/mapstructure/v2

The original one is no longer maintained. Viper is a very active project so it should be in good.